master - Add procedure for mass migration using UI (#4895)
uyuni-project/uyuni-docs: modules/client-configuration/pages/client-upgrades-product-migration.adoc

== Product Mass Migration

> If you want to migrate a large number of clients to the next SP version, you can use {productname} API calls or System Set Manager (SSM)

Author: ```suggestion
If you want to migrate a large number of clients to the next SP version, you can use {productname} API calls or System Set Manager (SSM).
```

> If you want to migrate a large number of clients to the next SP version, you can use {productname} API calls or System set manager (SSM).

Reviewer: ```suggestion
If you want to migrate a large number of clients to the next SP version, you can use {productname} API calls or System Set Manager (SSM)
```

The [command]``spacecmd`` commandline tool provides a [systemitem]``system_scheduleproductmigration`` sub command, which can be used to schedule a migration for a large number of clients to the next minor version.

=== Perform a Product Mass Migration using System Set Manager (SSM)

.Procedure: Performing a Product Mass Migration using System Set Manager (SSM)
[role=procedure]
____

> . Add the systems you want to migrate to the System set manager (SSM) as described in xref:client-configuration:system-groups.adoc[].

Author: ```suggestion
. Add the systems you want to migrate to the System Set Manager (SSM) as described in xref:client-configuration:system-groups.adoc[].
```

. In the {productname} {webui}, navigate to menu:Systems[System Set Manager].
. Navigate to the [guimenu]``Product Migration`` tab. 
. Choose a target product and then click btn:[Select Channels].
. On the next page, click btn:[Schedule Migration].

> . If everything is correct, click btn:[Migrate] to start the migration or btn:[Dry Run] to do a dry run first.

Reviewer: Best to lead by recommending Dry Run first.

Author: ```suggestion
. If you want, click btn:[Dry Run] to do a dry run first, and if everything is correct, click btn:[Migrate] to start the migration.
```
